libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -109,6 +111,12 @@ public static List<String> extractTxnIdsFromCredentials(List<SignedJWT> credenti
         return txnIds;
     }
 
+    public static boolean isOperationalProfileVc(SignedJWT credential) throws ParseException {
+        var credVot = credential.getJWTClaimsSet().getStringClaim(VOT_CLAIM_NAME);
+        return credVot != null
+                && Vot.valueOf(credVot).getProfileType().equals(ProfileType.OPERATIONAL_HMRC);
```

Comment:
(We should be careful that there'll never be any unexpected VOT values, or this will blow up, but at the moment that shouldn't be possible - all the possible VOTs are described in the VTM document)

Reply:
So 'vot' can only have values specified in 'vtm'

Reply:
Yeah - they're actually published at https://oidc.account.gov.uk/trustmark

(This will need updating for inherited ids, which is a job for orchestration)